src/Integration/Flysystem/S3/tests/version-bridge-tests.php

```diff
@@ -17,3 +17,27 @@ class FilesystemAdapterTestCase extends TestCase
         }
     }
 }
+
+namespace {
+    if (!function_exists('reset_function_mocks')) {
```

Comment:
because of https://github.com/thephpleague/flysystem/commit/a5735c905f5f86c47b8073f75c33b5fd2023288d#r37884566